.github/workflows/test.yml

```diff
@@ -56,17 +56,19 @@ jobs:
           python3 -m pip install -vvv .[tests]
 
       - name: Run Serial Tests
+        working-directory: ./tests
         run: |
           export OMP_NUM_THREADS=2
           export OPENBLAS_NUM_THREADS=2
           export MPI_DISABLE=1
-          python3 -m unittest discover
+          python3 -m pytest
```

Comment:
Can you add `-vv` after pytest? The increased verbosity makes immediately clear which test exactly failed.

https://docs.pytest.org/en/stable/how-to/output.html#verbosity